text2term/term_collector.py

```diff
@@ -336,6 +338,9 @@ def _load_ontology(self, ontology_iri):
         """
         self.logger.info("Loading ontology %s...", ontology_iri)
         start = time.time()
+        owl_link = bioregistry.get_owl_download(ontology_iri)
```

Comment:
the `bioregistry.get_owl_download()` function is supposed to take an ontology prefix, maybe this is a case where you can wrap the original function with a convenience function to make it more clear when you're supposed to use a prefix vs a full IRI